tests/e2e/test_rosacli_node_pool.go

```diff
@@ -1777,4 +1780,105 @@ var _ = Describe("Edit nodepool",
 							"ERR: Expected a valid http tokens value : ec2-metadata-http-tokens " +
 								"value should be one of 'required', 'optional'"))
 			})
+
+		It("Edit machine pool to the ROSA Hypershift cluster - [id:56778]", labels.Runtime.Day2, labels.High,
+			func() {
+				var (
+					replicas         string
+					machinepoolNames = []string{common.GenerateRandomName("np", 4), common.GenerateRandomName("np", 4)}
+					np               *rosacli.NodePool
+				)
+
+				By("List Machinepools")
+				machinePoolList, err := rosaClient.MachinePool.ListAndReflectNodePools(clusterID)
+				Expect(err).ToNot(HaveOccurred())
+				if profile.ClusterConfig.MultiAZ {
+					np = machinePoolList.Nodepool("workers-0")
+				} else {
+					np = machinePoolList.Nodepool("workers")
+				}
+				Expect(np).ToNot(BeNil())
+				for _, npName := range machinepoolNames {
+					By("Create a Machinepool")
+					taints := "k56778=v:NoSchedule,k56778-2=:NoExecute"
+					labels := "test56778="
+					replicas = "3"
+					output, err := rosaClient.MachinePool.CreateMachinePool(clusterID, npName,
+						"--replicas", replicas,
+						"--labels", labels,
+						"--taints", taints,
+						"--enable-autoscaling=false",
+						"--autorepair=false",
+					)
+					Expect(err).ToNot(HaveOccurred())
+					Expect(rosaClient.Parser.TextData.Input(output).Parse().Tip()).
+						Should(ContainSubstring(
+							"Machine pool '%s' created successfully on hosted cluster '%s'",
+							npName,
+							clusterID))
+
+					By("Verify edited machinepool")
+					npList, err := rosaClient.MachinePool.ListAndReflectNodePools(clusterID)
+					Expect(err).ToNot(HaveOccurred())
+					np = npList.Nodepool(npName)
+					Expect(np).ToNot(BeNil())
+					Expect(np.AutoScaling).To(Equal("No"))
+					Expect(np.Replicas).To(Equal("0/3"))
+					Expect(np.AutoRepair).To(Equal("No"))
+
+					By("Retrieve created Machinepool from list")
+					machinePoolList, err = rosaClient.MachinePool.ListAndReflectNodePools(clusterID)
+					Expect(err).ToNot(HaveOccurred())
+					np = machinePoolList.Nodepool(npName)
+					Expect(np).ToNot(BeNil())
```

Comment:
I think this step can be deleted. You do exactly the same in previous step

Reply:
Agree, deleted this part